sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -94,10 +94,7 @@ export interface EventDataAdapterParameters {
 // @public
 export interface EventDataBatch {
     readonly count: number;
-    // @internal
-    _generateMessage(): Buffer;
```

Comment:
hmm, I removed it from public surface as it's internal.  Not sure if I should bring it back.